apps/ett/e2e-ett/src/support/landing-page.po.ts

```diff
@@ -16,6 +16,7 @@
  */
 export const findProductLogo = () =>
   cy.findByRole('img', { name: 'EnergyOrigin' });
-export const findStartLink = () => cy.findByRole('link', { name: /Start/i });
+export const findStartLink = () =>
+  cy.get('eo-header').findByRole('link', { name: /Start/i });
```

Comment:
question: Why is it necessary to add the `eo-header` selector?

Reply:
The end to end tests are failing because we now have 3 identical "START" links on the landing page - Since they are all identical, and get the link to Signaturgruppen from the same Observable, I have limited the test to target just the start link in the eo-header selector. Cypress fails with multiple options/elements.

Reply:
issue: We should avoid using selectors in our tests in order to prefer using accessible navigation since this is what our users are able to do. Use `findAllByRole` and pick a query result instead of `cy.get` + `cy.findByRole`.